_haikus/Looking_for_connections.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Looking for connections
+author: Andrew
+---
+
+Clouds are light and white<br>
+Devices we look to connect<br>?
```

Comment:
This line is one syllable over, and the question mark should be before the `<br>` tag.

A suggestion:

```
Devices look to connect?<br>
```

Reply:
Thanks for the suggestion!!